CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Okta Python SDK Changelog
 
+## v2.0.0
+- Regenerate code using the [open API spec v2.5.0](https://github.com/okta/okta-management-openapi-spec/releases/tag/openapi-2.5.0).
+- Make sign_on_mode of all apps instance of enum, issue #198 (this change might be not compatible with custom clients code, although probability is low)
```

Comment:
Since this is a potential breaking change, I suggest adding a section for this and briefly show what changed (see [this](https://github.com/okta/okta-sdk-dotnet/blob/master/CHANGELOG.md#breaking-changes) for example).

Reply:
Added